worlds/openrct2/Options.py

```diff
@@ -0,0 +1,862 @@
+    roaring_twenties_prison_island = 116
+    rock_n_roll_flower_power = 117
+    dark_age_castle = 118
+    future_first_encounters = 119
+    mythological_animatronic_film_set = 120
+    jurassic_safari = 121
+    roaring_twenties_schneider_cup = 122
+    future_future_world = 123
+    mythological_cradle_of_civilization = 124
+    prehistoric_stone_age = 125
+    roaring_twenties_skyscrapers = 126
+    rock_n_roll_rock_n_roll = 127
+    alton_towers = 128
+    blackpool_pleasure_beach = 129
+    heide_park = 130
+    six_flags_belgium = 131
+    six_flags_great_adventure = 132
+    six_flags_holland = 133
+    six_flags_magic_mountain = 134
+    six_flags_over_texas = 135
+    fort_anachronism = 136
+    build_your_own_six_flags_belgium = 137
+    build_your_own_six_flags_great_adventure = 138
+    build_your_own_six_flags_holland = 139
+    build_your_own_six_flags_magic_mountain = 140
+    build_your_own_six_flags_park = 141
+    build_your_own_six_flags_over_texas = 142
+    random_RCT1 = 143
+    random_loopy_landscapes = 144
+    random_corkscrew_follies = 145
+    random_RCT2 = 146
+    random_wacky_worlds = 147
+    random_time_twister = 148
+    random_RCT1_expansions = 149
+    random_RCT2_expansions = 150
+    archipelago_madness_vanilla = 151
+    archipelago_madness_expansions = 152
+    
+class LocationBalancingMode(IntEnum):
+    disabled = 0
+    compromise = 1
+    full = 2
+
+class DeathLinkMode(IntEnum):
+    disabled = 0
+    enabled = 1
+
+class Visibility(IntEnum):
+    nothing = 0
+    recipient = 1
+    full = 2
+
+class Difficulty(IntEnum):
+    very_easy = 0
+    easy = 1
+    medium = 2
+    hard = 3
+    extreme = 4
+
+class Preferred_Intensity(IntEnum):
+    less_intense = 0
+    normal = 1
+    more_intense = 2
+
+class Randomization_Range(IntEnum):
+    none = 0
+    low = 1
+    medium = 2
+    high = 3
+    extreme = 4
+
+class Scenario_Length(IntEnum):
+    synchronous_short = 0
+    synchronous_long = 1
+    lengthy = 2
+    marathon = 3
+    
+class Stat_ReRolls(IntEnum):
+    never = 0
+    infrequent = 1
+    semi_frequent = 2
+    frequent = 3
+    very_frequent = 4
+    extremely_frequent = 5
+
+class Difficult_Guest_Generation(IntEnum):
+    off = 0
+    unlockable = 1
+    on = 2
+
+class Difficult_Park_Rating(IntEnum):
+    off = 0
+    unlockable = 1
+    on = 2
+
+class Forbid_High_Construction(IntEnum):
+    off = 0
+    unlockable = 1
+    on = 2
+
+class Forbid_Landscape_Changes(IntEnum):
+    off = 0
+    unlockable = 1
+    on = 2
+
+class Forbid_Marketing_Campaigns(IntEnum):
+    off = 0
+    unlockable = 1
+    on = 2
+
+class Forbid_Tree_Removal(IntEnum):
+    off = 0
+    unlockable = 1
+    on = 2
+
+class OpenRCT2OnToggle(DefaultOnToggle):
+    @property
+    def result(self) -> bool:
+        return self.value
+
+
+class OpenRCT2Toggle(Toggle):
+    @property
+    def result(self) -> bool:
+        return self.value
+
+
+class LocationBalancing(Choice):
```

Comment:
Where/How does this option work exactly? I don't see it used anywhere

Reply:
Welp. This is embarassing. When I first copied this over, I conflated "Progression Balancing" with "Location Balancing", and never looked at it again.